### DISCUSSION TITLE
How to track down decompiler errors when no output is produced

### DISCUSSION THREAD

**Discussion by Wall-AF** [NationalSecurityAgency/ghidra#5114] [General]
Having updated my local repo (commit d76f59ee69) a method that I haven't visited for a long while now produces `Low-level Error: Unsupported data-type for ResolveUnion`!

If I create a new function below the first couple of assembly statements (that just do `INC BP` followed by `PUSH BP`) at least the decompiler then produces something, but all stack context disappears.

Therefore how can anyone track down errors like these???

**Comment by dev747368** [comment]
did you rebuild the decompiler native executable?

**Reply by Wall-AF** [reply]
Yes
